assets/model_monitoring/components/model_monitor_action_analyzer/action_analyzer_identify_problem_traffic/spec.yaml

```diff
@@ -27,16 +27,10 @@ inputs:
     type: string
     optional: false
     description: model name, e.g., name of the deployment for Azure OpenAI
-  prompt_column_name:
+  show_topic_name:
```

Comment:
why don't we use bool type?

Reply:
bool type has some limitations in the yaml arguments. Fixed by a workaround.